src/main/kotlin/org/rust/cargo/project/workspace/CargoWorkspace.kt

```diff
@@ -78,10 +77,10 @@ class CargoWorkspace private constructor(
 
     private val targetByCrateRootUrl = packages.flatMap { it.targets }.associateBy { it.crateRootUrl }
 
-    fun findCrateByName(normName: String): Target? =
-        packages
-            .mapNotNull { it.libTarget }
-            .find { it.normName == normName }
+    fun findCrateByName(normName: String, inPackage: Package? = null): Target? =
```

Comment:
Hm, and it may be the case that we need `Target` and not the `Package` here. `dev-dependencies` affect only tests and build script, but not the main target.